Spectral/index.html

```diff
@@ -0,0 +1,192 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+</head>
+
+<body>
+    <header class="header-section">
+
+        <article class="header-card">
+            <section class="header-spectral">
+                <hr class="styled">
+                <h1>SPECTRAL</h1>
+                <hr class="styled">
+                <p>ANOTHER FINE RESPONSIVE</br>
+                    SITE TEMPLATE FREEBIE</br>
+                    CRAFTED BY <a href="#">HTML5 UP</a>.</p>
+                <a href="#" class="btn">Activate</a>
+            </section>
+        </article>
+
+    </header>
+
+    <main>
+        <article class="main-section">
+
+            <section class="main-section-card">
```

Comment:
instead of using main-section-card  class names use name like cards or card-container